spyder/plugins/outlineexplorer/tests/test_widgets.py

```diff
@@ -145,19 +145,18 @@ def _create_bot(code, filename):
         outlineexplorer.setFixedSize(400, 350)
 
         qtbot.addWidget(outlineexplorer)
-        return outlineexplorer, qtbot
-    return _create_bot
+        return outlineexplorer
+    return _create_outlineexplorer
```

Comment:
Is it better practice to ``yield`` then explicitly ``.close()`` the ``outlineexplorer`` widget here to ensure everything is properly cleaned up after the test in question? I've been wondering about this myself.